app/components/layouts/SideTasks.tsx

```diff
@@ -12,8 +12,8 @@ import { getAssignedTasks } from '@/services/backend/tasks';
 import { formatDate } from '@/utils/date';
 
 const taskTypeLabels = {
-  [TaskType.SIGN_MOU]: 'Sign MOU',
-  [TaskType.REVIEW_MOU]: 'Review MOU',
+  [TaskType.SIGN_PUBLIC_CLOUD_MOU]: 'Sign Private Cloud MOU',
```

Comment:
I believe here is also `Public cloud` `Sign Private Cloud MOU`

Reply:
fixed!